documentation/SupportingDocuments/Purpose Consent Proposal comparison.md

```diff
@@ -0,0 +1,35 @@
+
+
+
+# Purpose, API & open Gateway product granularity
+
+## Objectives of this document
+
+A lot of discussion around the concept of purpose and its management in CAMARA API occurred in particular in the Issue#32 (https://github.com/camaraproject/IdentityAndConsentManagement/issues/32).
+
+We have, as of now, 2 distinct proposal on the table: one initially provided by Telefonica, and then a second proposed by the Orange team. Both proposal have a lot of common & agreed on the strong requirement for purpose & consent management. But as both proposal diverge on the Open Gateway product whom frame the API commercialization, the following table sum-up main points of both proposal in a form of a FAQ.
+
+## Proposal presentation
+
+| Topic     | Telefonica proposal | Orange proposal |
+| --------- | ------------------- | --------------- |
+| What is the Open Gateway (OPGW) Product granularity? |  |  Open Gateway product matches 1-1 with a CAMARA API. For example, as of now, the CAMARA Device location API family features 2 APIs (2 yaml): `location-retrieval` and `location_verification` : we consider them as 2 separate OPGW product. The OPGW product covers completely the API (all the resources of the API). This is an important point because it means that API granularity must not be considered only from a technical aspect but also from a business aspect (which is already the case for CAMARA API). <br> The OPGW product is **not** strong-coupled with a purpose but independent of it. However in the catalog it is possible to recommend OPGW product with purpose (see next point).|

```

Comment:
Telefónica disagrees. For example, the SIM Swap API has two different endpoints (/check and /retrieve-date) and these endpoints will be different products as they require different legal basis. There will not be one SIM Swap product for the whole API (all resources of the API).